File: app/src/main/java/com/testndk/jnistudy/ui/activity/BsdiffActivity.java
package com.testndk.jnistudy.ui.activity;

import android.view.View;

import com.testndk.jnistudy.R;
import com.testndk.jnistudy.ui.bsdiff.BsdiffUtilsJava;

public class BsdiffActivity extends BaseActivity {
    @Override
    public int initLayout() {
        return R.layout.activity_bsdiff;
    }

    public void onClickMerge(View view) {
        BsdiffUtilsJava.mergeApk(this);
    }
}

File: app/src/main/java/com/testndk/jnistudy/ui/activity/BsdiffActivity.kt
package com.testndk.jnistudy.ui.activity

import android.view.View
import com.testndk.jnistudy.R
import com.testndk.jnistudy.ui.BaseActivity
import com.testndk.jnistudy.ui.bsdiff.BsdiffUtilsJava

class BsdiffActivity : BaseActivity() {
   override fun initLayout() = R.layout.activity_bsdiff

   fun onClickMerge(view: View?) {
       BsdiffUtilsJava.mergeApk(this)
   }
}
